tests/test_auth_app.py

```diff
@@ -14,8 +14,8 @@ class TestAuthApp(unittest.TestCase):
     def test_auth_success(self, requests):
         requests.post.return_value = MockResponse(200, create_mock_json("tests/resources/auth_app_success.json"))
         api = TaigaAPI(host="host")
-        api.auth_app("valid-app-id", "valid-app-secret", "valid-auth-code", "valid-state")
-        self.assertEqual(api.token, "f4k3")
```

Comment:
@heck-gd Same as above: shouldn't we check that `api.token` is correct after doing `api.auth_app`?